content/en/docs/resources/demos-talks-posts/articles.md

```diff
@@ -29,7 +29,6 @@ weight: 10
 * [Jenkins X: Preview Environment](https://medium.com/@MichalFoksa/jenkins-x-preview-environment-3bf2424a05e4) by [Michal Foksa](https://medium.com/@MichalFoksa), January 10th, 2019
 * [Continuous Delivery with Amazon EKS and Jenkins X](https://aws.amazon.com/blogs/opensource/continuous-delivery-eks-jenkins-x/) by [Henryk Konsek](https://twitter.com/hekonsek), 7th November, 2018
 * [Trying out Jenkins X on Azure Kubernetes Service(AKS)](https://medium.com/@puichan/trying-out-jenkins-x-on-azure-kubernetes-service-aks-20fe94bd72d2) by [PC Chan](https://medium.com/@puichan) on October 29th, 2018
-* [SetUp Jenkins X on Minikube kubernetes Cluster](https://medium.com/@mayank.023/setup-jenkinsx-on-minikube-kubernetes-cluster-c519264bb991) by [mayank verma](https://medium.com/@mayank.023) on September 30th, 2018  
```

Comment:
Not sure if we want to remove references to talks about minikube or just put a comment that minikube is not supported.